xfce/orage/Portfile

```diff
@@ -14,9 +14,11 @@ long_description ${description}
 homepage        https://www.xfce.org
 master_sites    http://archive.xfce.org/src/apps/${name}/${branch}/
 use_bzip2       yes
-checksums       md5 2b7f5d38cb5c6edbcc65d0f52a742e46 \
-                sha1 c856ad30f7a17a732d5cc794ef90aedf98575699 \
-                sha256 3cf9aa441ae83c8688865f82217025cdf3ebaa152cce4571777b8c2aa8dd9062
+checksums       md5     c12765da61022c710bb0d5aab3c9c56f \
+                sha1    d346563413ec6a3aa45e93899279e226f0a3ecdf \
```

Comment:
We no longer use md5 or sha1 checksums in ports. It is sufficient to give the others. Please fix.

Reply:
Fixed.